src/main/resources/application.conf

```diff
@@ -224,6 +224,10 @@ mantis {
     # Doc: https://github.com/ethereum/EIPs/issues/160
     eip160-block-number = "3000000"
 
+    # EIP-161 fork block number (ETH Only)
+    # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-161.md
+    eip161-block-number = "9223372036854775807"
```

Comment:
what's that number?

Reply:
It is in: https://github.com/input-output-hk/mantis/pull/306/files#diff-1494bba019381b0106c673078b722f7cL21, and it is 2675000.
So it is only enabled in ETH network, setting it `Long.maxValue` here disables it for other networks.

Reply:
:+1:

Reply:
What about setting it to `null` and when reading config mapping it to an `Option` instead of `Long.maxValue`?

Reply:
Actually, other clients use both of these strategies (empty values or large block numbers) to disable correct EIPs. During pr to EIP106 (https://github.com/input-output-hk/mantis/blob/ea26ee798ed73b9973a2d8eca3abca1db410c9b9/src/main/resources/application.conf#L211)  we agreed with @rtkaczyk to use higher block number and here it is just a matter of consistency.

Reply:
Yeah, but `9223372036854775807` is really random looking number and as we can see it raises confusion. Could we use 10000...000 instead, or accept hexadecimal values and use 0x7fffffffffffffff (parity or geth uses that).

Reply:
Oh we should expect that users that will look at configuration should at first glance recognize that
`9223372036854775807` is really `Long.maxValue`  😄  and more seriously I would go with
`10000...000` because most of our config are just decimal values, and this hex would like out of place for me